src/map/script.c

```diff
@@ -6397,6 +6530,89 @@ static BUILDIN(callfunc)
 
 	return true;
 }
+
+static BUILDIN(callfunctionofnpc) {
```

Comment:
Please add a documentation. (Switching between code and script_commands.txt is a pain in the arse. :smile:)